filter/unquote_resolvers.py

```diff
@@ -0,0 +1,79 @@
+from typing import Any, Dict, List, Union
+
+import yaml
+from yaml.nodes import ScalarNode
+
+try:
+    from yaml import CSafeDumper as SafeDumper
+except ImportError:
+    from yaml import SafeDumper  # type: ignore
+
+
+class CustomDumper(SafeDumper):
+    pass
+
+
+def unquoted_tag_representer(dumper: CustomDumper, data: "Tag") -> ScalarNode:
+    return dumper.represent_scalar(data.tag, data.value, style="")
+
+
+class Tag:
+    def __init__(self, tag: str, value: str) -> None:
+        self.tag = tag
+        self.value = value
+
+
+CustomDumper.add_representer(Tag, unquoted_tag_representer)
+
+
+def unquote_resolvers(
+    data: Union[Dict, List],
+    indent: int = 2,
+    output_indent: int = 0,
+    *args: Any,
+    **kw: Any
+) -> str:
+    """
+    Remove quotes around resolver expressions and
+    produce nice YAML.
```

Comment:
You should also describe the parameters on there and what they do.

Reply:
done